### ISSUE TITLE
Manual backup setting is not persistent

### ISSUE THREAD

**Issue by kssi** [Drive4ik/simple-tab-groups#459]
**Describe the bug**

The following setting for _manual_ backups is not persistent:

- Include tab thumbnails in backup (makes backup file size bigger)

I have to check the box every time I visit the settings panel.

**To Reproduce**

1. Enable the aforementioned setting
2. Leave the settings panel (close the tab)
3. Go back to the settings panel
4. The check-mark has disappeared 

**Expected behavior**

I can't think of a valid reason for this behavior to be intentional.

**Desktop**

 - OS and version: macOS El Capitan 10.11.6
 - Firefox version: 68.0.2 (64-bit)
 - Simple Tab Groups version: 4.0.1.2

**Comment by Drive4ik** [comment]
Yes that's right. Only this is not a bug, I just found it unnecessary to save these 2 checkmarks. They are not used anywhere else except on this page. And to press it is not a big deal. It’s just that they don’t carry any payload .. but they’ll just be in the settings .. But with automatic backup, the state of the ticks is saved so it is important.

**Comment by kssi** [comment]
Problem is:
- This is not the behavior one would reasonably expect,
- This makes the whole user interface inconsistent (since some random settings are arbitrarily reset to their default and some others are not).

So it does appear as a bug (and it's definitely disturbing as far as I'm concerned).

I can understand why this is not a priority, but this should be fixed IMHO. User settings are persistent in essence.

**Comment by Drive4ik** [comment]
Let me do it - neither to you nor to me))
I will do a save, but only in localStorage. These fields will be reset when reinstalling the addon. That is, these settings will not get into the backup file, and of course they will not be restored from there. I do not want to pile up extra settings...
? :)
